### PR TITLE
Update br.m3u

### PR DESCRIPTION
**Pull request by kimbrasil**

### CODE REVIEW

File: channels/br.m3u [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Agro Brasil TV
+http://45.162.230.234:1935/agrobrasiltv/agrobrasiltv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/vpzYZDT.png" group-title="",ALL Sports
+https://5cf4a2c2512a2.streamlock.net/dgrau/dgrau/chunklist.m3u8
+#EXTINF:-1 tvg-id="band.br" tvg-name="Band" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/h8drbLl.png" group-title="",Band
+http://evpp.mm.uol.com.br:1935/geob_band/app/playlist.m3u8
+#EXTINF:-1 tvg-id="bandnews.br" tvg-name="Band News" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/W5xuts0.png" group-title="",Band News
+https://evpp.mm.uol.com.br/geob_band/bandnewstv/chunklist_w5392006.m3u8
+#EXTINF:-1 tvg-id="bandsports.br" tvg-name="Band Sports" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/TG25xuW.png" group-title="",Band Sports
+https://evpp.mm.uol.com.br/geob_band/bandsports/playlist.m3u8
```
Comment (freearhey) [inline channels/br.m3u:13]: Links does not work. 
Tested in Elmedia Video Player, VLC Player and streamtest.in

File: channels/br.m3u [hunk 2]
```diff
@@ -9,12 +21,50 @@ https://5a7bcfb834235.streamlock.net/fiocruz/fiocruz/playlist.m3u8?DVR
...
+https://medias.sgr.globo.com/hls/vCBNSP/vCBNSP.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",CBN Rio
+https://medias.sgr.globo.com/hls/vCBNRJ/vCBNRJ.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="http://i.imgur.com/SYPYeHt.jpg" group-title="",Cine+
 http://189.86.89.116/hls-live/livecm/_definst_/liveevent/livestream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",COM Brasil
+#EXTINF:-1 tvg-id="cnt.br" tvg-name="CNN" tvg-language="Portuguese" tvg-logo="" group-title="",CNN Brasil
+https://streaming.cnnbrasil.com.br:443/cnndigital_mainOutput2.m3u8
+#EXTINF:-1 tvg-id="cnt.br" tvg-name="CNT" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/GVu2RbW.png" group-title="",CNT
+https://cdn0.showcase.com.br/hls/cnt/cwb.m3u8
```
Comment (freearhey) [inline channels/br.m3u:33]: Links does not work.
Tested in Elmedia Video Player, VLC Player and streamtest.in

File: channels/br.m3u [hunk 3]
```diff
@@ -9,12 +21,50 @@ https://5a7bcfb834235.streamlock.net/fiocruz/fiocruz/playlist.m3u8?DVR
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Disney XD
+http://playplusdisney-lh.akamaihd.net/i/pp_dsneyxd@376337/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/QusWJuw.png" group-title="",ESPN
+https://playplusespn-lh.akamaihd.net/i/pp_espn@374460/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/H84bwyI.png" group-title="",ESPN 2
+http://playplusespn-lh.akamaihd.net/i/pp_espnmais@374460/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/t9ZZhSI.png" group-title="",ESPN Brasil
+https://playplusespn-lh.akamaihd.net/i/pp_espnbra@374460/index_720_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/VS0JbNj.png" group-title="",ESPN Extra
+http://playplusespn-lh.akamaihd.net/i/pp_espnext@374460/index_720_av-p.m3u8
```
Comment (freearhey) [inline channels/br.m3u:53]: Links does not work.
Tested in Elmedia Video Player, VLC Player and streamtest.in

File: channels/br.m3u [hunk 4]
```diff
@@ -23,22 +73,56 @@ https://cdn.jmvstream.com/w/inpara/_definst_/inpara/chunklist_w1305866021.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Record News
+https://playplusnews-lh.akamaihd.net/i/pp_nws@377849/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Record Pará
+https://playpluspa-lh.akamaihd.net/i/pp_pa@377468/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Record Rio
+https://playplusrjo-lh.akamaihd.net/i/pp_rj@377859/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Record Rio Grande do Sul
+https://playpluspoa-lh.akamaihd.net/i/pp_poa@377864/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Record São Paulo
+https://playplusspo-lh.akamaihd.net/i/pp_sp@350176/master.m3u8
```
Comment (freearhey) [inline channels/br.m3u:111]: Links does not work.
Tested in Elmedia Video Player, VLC Player and streamtest.in

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the links again and update the playlist if necessary so we can approve it.

File: channels/br.m3u (hunk 1 above)
Comment (kimbrasil) [reply channels/br.m3u:13]: Seems that these links are geo-restricted to Brazil.

File: channels/br.m3u (hunk 3 above)
Comment (kimbrasil) [reply channels/br.m3u:53]: Seems that these links are geo-restricted to Brazil.

File: channels/br.m3u (hunk 2 above)
Comment (kimbrasil) [reply channels/br.m3u:33]: Seems that these links are geo-restricted to Brazil.

File: channels/br.m3u (hunk 3 above)
Comment (SileC) [reply channels/br.m3u:53]: Yes, they are geo. Can you add a (GEO) at end of channel name?

Comment (freearhey) [reply channels/br.m3u:53]: We don't add geo-blocked channels to playlists because they're too hard to maintain.

Comment (kimbrasil) [reply channels/br.m3u:53]: So I'll remove this commit and make a fork for local usage, TY @freearhey  and @SileC !

Comment (SileC) [reply channels/br.m3u:53]: Will the fork be visible also in github/iptv or local only? Sorry, I have no idea how this works.

Comment (kimbrasil) [reply channels/br.m3u:53]: > Will the fork be visible also in github/iptv or local only? Sorry, I have no idea how this works.
When someone propose a change in a github project where don't is a moderator or admin, github ask if you wanna create a fork and propose the changes to the original project. Soon as possible I'll adjust my fork for use with georestricted channels to use locally.

**Pull request closed without merging**